packages/loader/driver-utils/src/stringToBuffer.ts

```diff
@@ -0,0 +1,35 @@
+/*!
```

Comment:
Given you are going in small PR chunks, you could skip adding this file, and instead bump common-util verison in next PR to make you changes in common-util available to all the code.